src/connectedk8s/azext_connectedk8s/tests/latest/test_connectedk8s_scenario.py

```diff
@@ -514,3 +515,49 @@ def test_troubleshoot(self,resource_group):
         # delete the kube config
         os.remove("%s" % (_get_test_data_file(managed_cluster_name + '-config.yaml')))
 
+    @live_only()
+    @ResourceGroupPreparer(name_prefix='conk8stest', location='eastus2euap', random_name_length=16)
```

Comment:
May I ask why you mark this test as `@live_only()`?

Reply:
All of the tests for our connectedk8s CLI are currently marked as live_only because these tests would not run as recording mode test or integration test because in out connectedk8s connect command in some steps we use some cache to get some values and in recording mode the cache doesnt get recorded and hence we decided to use live_mode only so that we can run the CLI testing pipeline whenever required in live mode only

Reply:
Got it, thanks~